utils/doclint/cli.js

```diff
@@ -31,7 +31,10 @@ const RED_COLOR = '\x1b[31m';
 const YELLOW_COLOR = '\x1b[33m';
 const RESET_COLOR = '\x1b[0m';
 
-run();
+run().catch(e => {
+  console.error(e && e.stack ? e.stack : e);
```

Comment:
doesn't `console.error(new Error())` already print the stack automatically to the console?

Reply:
yep, I've spent too much time with jest's serializable errors.